src/app/events/event-organize/event-organize.html

```
@@ -57,14 +57,34 @@ <h3 class="title-medium">
+                      ESNcard discount
+                    </span>
+                  }
+                  @if (user.checkedIn) {
+                    <span class="rounded bg-surface px-2 py-1 text-xs">
+                      Checked in
+                    </span>
+                  }
+                </div>
+                @if (!registrationOption.organizingRegistration) {
+                  <button
+                    mat-stroked-button
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)